org.KangLin.Rabbit.RemoteControl.json

```diff
@@ -0,0 +1,246 @@
+            "builddir": true,
+            "config-opts": [
+                "-Dexamples=false",
+                "-Dutils=false"
+            ]
+        },
+        {
+            "name": "FreeRDP",
+            "disabled": false,
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://github.com/FreeRDP/FreeRDP/archive/refs/tags/3.14.0.tar.gz",
+                    "sha256": "a7a979382703c2bc18aa9b22cfa12d7a441b6927ab45ac62ef8353786bc92241"
+                }
+            ],
+            "buildsystem": "cmake",
+            "builddir": true,
+            "config-opts": [
+                "-DCMAKE_VERBOSE_MAKEFILE=OFF",
+                "-DCMAKE_BUILD_TYPE:STRING=Release",
+                "-DCHANNEL_TSMF:BOOL=OFF",
+                "-DBUILD_TESTING:BOOL=OFF",
+                "-DWITH_MANPAGES:BOOL=OFF",
+                "-DWITH_WAYLAND:BOOL=OFF",
+                "-DWITH_GSSAPI:BOOL=OFF",
+                "-DWITH_SAMPLE:BOOL=OFF",
+                "-DWITH_OSS:BOOL=OFF",
+                "-DCHANNEL_URBDRC:BOOL=OFF",
+                "-DWITH_KRB5=ON"
+            ],
+            "cleanup": [
+                "/bin",
+                "/share"
+            ]
+        },
+        {
+            "name": "libvncserver",
+            "disabled": false,
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://github.com/LibVNC/libvncserver/archive/refs/tags/LibVNCServer-0.9.15.tar.gz",
+                    "sha256": "62352c7795e231dfce044beb96156065a05a05c974e5de9e023d688d8ff675d7"
+                }
+            ],
+            "buildsystem": "cmake",
+            "builddir": true,
+            "config-opts": [
+                "-DWITH_OPENSSL=ON",
+                "-DWITH_GCRYPT=OFF",
+                "-DWITH_EXAMPLES=OFF",
+                "-DWITH_TESTS=OFF"
+            ],
+            "cleanup": [
+                "/bin/*",
+                "/lib/libvncserver.so*"
+            ]
+        },
+        {
+            "name": "RabbitCommon",
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://github.com/KangLin/RabbitCommon.git",
+                    "branch": "master"
+                }
+            ],
+            "buildsystem": "simple",
+            "build-commands": [
+                "cp -R /run/build/RabbitCommon /var/tmp/"
+            ]
+        },
+        {
+            "name": "RabbitRemoteControl",
+            "disabled": false,
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://github.com/KangLin/RabbitRemoteControl.git",
+                    "branch": "master"
+                }
+            ],
+            "buildsystem": "cmake",
+            "builddir": false,
+            "config-opts": [
+                "-DRabbitCommon_ROOT=/var/tmp/RabbitCommon",
```

Comment:
this does not seem correct? why are you setting it to /var/tmp?

Reply:
see https://github.com/flathub/flathub/pull/6281#discussion_r1995464817